testing/chain_test.go

```diff
@@ -1,38 +1,28 @@
 package ibctesting_test
 
-import (
-	"testing"
+// func TestChangeValSet(t *testing.T) {
+// 	coord := ibctesting.NewCoordinator(t, 2)
+// 	chainA := coord.GetChain(ibctesting.GetChainID(1))
+// 	chainB := coord.GetChain(ibctesting.GetChainID(2))
 
-	"github.com/stretchr/testify/require"
+// 	path := ibctesting.NewPath(chainA, chainB)
+// 	coord.Setup(path)
 
-	sdk "github.com/cosmos/cosmos-sdk/types"
-	"github.com/cosmos/cosmos-sdk/x/staking/types"
-	ibctesting "github.com/cosmos/ibc-go/v3/testing"
-)
+// 	amount, ok := sdk.NewIntFromString("10000000000000000000")
+// 	require.True(t, ok)
+// 	amount2, ok := sdk.NewIntFromString("30000000000000000000")
+// 	require.True(t, ok)
 
-func TestChangeValSet(t *testing.T) {
-	coord := ibctesting.NewCoordinator(t, 2)
-	chainA := coord.GetChain(ibctesting.GetChainID(1))
-	chainB := coord.GetChain(ibctesting.GetChainID(2))
+// 	val := chainA.App.GetStakingKeeper().GetValidators(chainA.GetContext(), 4)
 
-	path := ibctesting.NewPath(chainA, chainB)
-	coord.Setup(path)
+// 	chainA.App.GetStakingKeeper().Delegate(chainA.GetContext(), chainA.SenderAccounts[1].SenderAccount.GetAddress(),
+// 		amount, types.Unbonded, val[1], true)
+// 	chainA.App.GetStakingKeeper().Delegate(chainA.GetContext(), chainA.SenderAccounts[3].SenderAccount.GetAddress(),
+// 		amount2, types.Unbonded, val[3], true)
 
-	amount, ok := sdk.NewIntFromString("10000000000000000000")
-	require.True(t, ok)
-	amount2, ok := sdk.NewIntFromString("30000000000000000000")
-	require.True(t, ok)
+// 	coord.CommitBlock(chainA)
 
-	val := chainA.App.GetStakingKeeper().GetValidators(chainA.GetContext(), 4)
-
-	chainA.App.GetStakingKeeper().Delegate(chainA.GetContext(), chainA.SenderAccounts[1].SenderAccount.GetAddress(),
-		amount, types.Unbonded, val[1], true)
-	chainA.App.GetStakingKeeper().Delegate(chainA.GetContext(), chainA.SenderAccounts[3].SenderAccount.GetAddress(),
-		amount2, types.Unbonded, val[3], true)
-
-	coord.CommitBlock(chainA)
-
-	// verify that update clients works even after validator update goes into effect
-	path.EndpointB.UpdateClient()
-	path.EndpointB.UpdateClient()
-}
+// 	// verify that update clients works even after validator update goes into effect
+// 	path.EndpointB.UpdateClient()
+// 	path.EndpointB.UpdateClient()
+// }
```

Comment:
I'd prefer to delete chain_test.go and make an issue reminding to reintroduce the tesst, with a link to the commit containing the code, instead of commenting out.